test/components/connect.spec.js

```diff
@@ -1118,7 +1139,6 @@ describe('React', () => {
           expect(fetchSpy.calls.length).toBe(2)
 
           // set BAR again, but will not be refetched
-          // TODO: no need to re-render here
```

Comment:
I removed these because it's no longer true, there is a need to re-render there because the prop changes in a way that is not shallow equal